package.json

```diff
@@ -14,35 +14,35 @@
   },
   "main": "index.js",
   "scripts": {
-    "start": "NODE_ENV=production node --trace-warnings ./src/server/server.babel.js",
+    "start": "NODE_ENV=${NODE_ENV:-production} node --trace-warnings ./src/server/server.babel.js",
     "start:tunnel": "export $(grep ^ULTRAHOOK_API_KEY .env | tr -d '\"') && ultrahook -k $ULTRAHOOK_API_KEY dev 3000",
-    "dev": "if [ ! -f \"dll/vendors.json\" ] ; then npm run build:dll ; fi ; NODE_ENV=development node --trace-warnings ./src/server/server.babel.js",
```

Comment:
[FYI] - most of these changes are just me re-ordering things in rough alphabetical order / trying to keep similar scripts close to each other.